server/src/http_server/cmd.rs

```diff
@@ -17,13 +16,20 @@ pub(crate) async fn serve() -> Result<()> {
     .into_diagnostic()?;
 
     info!("Spawning Tasks");
-    let futures = vec![
+    let mut futures: Vec<tokio::task::JoinHandle<std::prelude::v1::Result<(), miette::Error>>> = vec![
         tokio::spawn(run_server(
             routes::make_router(syntax_css).with_state(app_state.clone()),
         )),
         tokio::spawn(job_worker(app_state.clone(), job_registry)),
-        tokio::spawn(run_cron(app_state.clone())),
     ];
+    
+    if std::env::var("CRON_DISABLED").unwrap_or_else(|_| "false".to_string()) != "true" {
```

Comment:
This code checks an environment variable to determine whether to `enable `or` disable cron `jobs.